### PR TITLE
Christy/issue212

### PR DESCRIPTION
**Pull request by creynolds75**
Added user-select settings to SCSS files

### CODE REVIEW

**Comment by veggiesaurus** [comment]
@kswang1029 can you test this and see if it solves #212 without interfering with normal functionality?

**Comment by kswang1029** [comment]
Overall I don't see something odd. 
Just a suggestion: it might be still good to enable highlight and copy-paste of the cursor info bar itself on the image viewer. I found it is handy to be able to copy coordinates or pixel value, frequency or velocity.

**Comment by veggiesaurus** [comment]
> Overall I don't see something odd.
> Just a suggestion: it might be still good to enable highlight and copy-paste of the cursor info bar itself on the image viewer. I found it is handy to be able to copy coordinates or pixel value, frequency or velocity.

Agreed, @ChristyReynolds please add this. 

Also, the dialogs are still behaving incorrectly: 
image

File: src/components/Stats/#StatsComponent.scss# [hunk 1]
```diff
@@ -0,0 +1,102 @@
+\1;95;0c@import "../../../node_modules/@blueprintjs/core/lib/scss/variables.scss";
```
Comment (veggiesaurus) [inline src/components/Stats/#StatsComponent.scss#]: not sure what this file is for

**Comment by creynolds75** [comment]
Thanks Angus!

**Pull request merged**